Resources/views/CRUD/list_email.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field%}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- elseif field_description.options.as_string is defined and field_description.options.as_string -%}
+        {{ value }}
+    {%- else -%}
+        {%- set url = '' -%}
+        {%- if field_description.options.subject is defined and field_description.options.subject is not empty -%}
+            {%- set subject = field_description.options.subject -%}
+            {%- set url = 'subject=' ~ subject -%}
```

Comment:
Feel free to push your proposition! :+1: 

I suggest you to make it on a separate commit. It would be easier for you to remove it if we decide for another solution.